SeriesGuide/src/main/java/com/battlelancer/seriesguide/ui/dialogs/AddShowDialogFragment.java

```diff
@@ -169,6 +180,11 @@ public void onClick(View v) {
         mButtonPositive.setOnClickListener(new OnClickListener() {
             @Override
             public void onClick(View v) {
+                if(mAdapterCallback != null) {
+                    mShow.isAdded = true;
+                    mAdapterCallback.notifyDataSetChanged();
```

Comment:
It's probably better to post an event here that the relevant `TraktAddFragment` then reacts to.